docs/source/notebooks/index.md

```diff
@@ -1,5 +1,5 @@
-(clv_examples)=
-# Examples
+(howto)=
+# How to?
```

Comment:
Thoughts on without How to without question mark?
ie.
- How to 
- How-to

Reply:
+1